.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         path: |
           dist/jitsi-meet-amd64.deb
           dist/jitsi-meet-x86_64.AppImage
+          dist/jitsi-meet-arm64.AppImage
```

Comment:
Shall we also add the deb?

Reply:
Check if that works as cross-packaging under amd64

Reply:
Local tests look good, I'll give it a try